app/tests/src/ingestion/test_pdf_postprocess.py

```diff
@@ -144,32 +152,47 @@ def test_concatenate_list_items():
         EnrichedText(
             text="Introduction.",
             type=TextType.NARRATIVE_TEXT,
-            headings=[Heading(title="Overview", level=1)],
+            headings=[Heading(title="Overview", level=1, pageno=None)],
             page_number=1,
+            id=None,
+            stylings=None,
+            links=None,
         ),
         EnrichedText(
             text="First item.\nSecond item.",
             type=TextType.LIST,
-            headings=[Heading(title="Overview", level=1)],
+            headings=[Heading(title="Overview", level=1, pageno=None)],
             page_number=1,
+            id=None,
+            stylings=None,
+            links=None,
         ),
         EnrichedText(
             text="Another narrative text.",
             type=TextType.NARRATIVE_TEXT,
-            headings=[Heading(title="Overview", level=1)],
+            headings=[Heading(title="Overview", level=1, pageno=None)],
             page_number=1,
+            id=None,
+            stylings=None,
+            links=None,
         ),
         EnrichedText(
             text="Narrative starting a new list: \nFirst item in new list.\nSecond item in new list.",
             type=TextType.LIST,
-            headings=[Heading(title="Overview", level=1)],
+            headings=[Heading(title="Overview", level=1, pageno=None)],
             page_number=1,
+            id=None,
+            stylings=None,
+            links=None,
         ),
         EnrichedText(
             text="New list item in new section with continuing sentence on next page",
             type=TextType.LIST_ITEM,
-            headings=[Heading(title="Section 1", level=1)],
+            headings=[Heading(title="Section 1", level=1, pageno=None)],
```

Comment:
Headings should all have page numbers
```suggestion
            headings=[Heading(title="Section 1", level=1, pageno=1)],
```

Reply:
Reminder to replace all occurrences of `pageno=None` above with `pageno=1`.